CHANGELOG.rst

```diff
@@ -11,6 +11,15 @@ Minor
 * Remove dependence on all ``source_stream`` APIs except for ``read()``.
   `#103 <https://github.com/aws/aws-encryption-sdk-python/issues/103>`_
 
+Potentially Breaking
```

Comment:
Minor quibble: I might call this "potentially backwards incompatible" instead, but not a strong preference.

Reply:
Sure.